hipposerve/api/app.py

```diff
@@ -5,21 +5,26 @@
 from contextlib import asynccontextmanager
 
 from beanie import init_beanie
-from fastapi import FastAPI, Request
+from fastapi import Depends, FastAPI, Request
+from fastapi.exceptions import RequestValidationError
 from fastapi.responses import RedirectResponse
 from loguru import logger
 from motor.motor_asyncio import AsyncIOMotorClient
 from starlette.datastructures import URL
+from starlette.exceptions import HTTPException
```

Comment:
Don't be chat gptting imports :).

I would recommend importing HTTPException from fastapi.

FWIW for URL parsing I would also rather use the builtin urllib parse and unparse functions.

Reply:
I actually got [the import from FastAPI's error handling docs].(https://fastapi.tiangolo.com/tutorial/handling-errors/#override-request-validation-exceptions). I first tried to import it from FastAPI and it didn't work but something else may have been amiss!

And I'll look into grabbing the id with the builtin `urllib` rather than pulling it off the Starlette request object.